### ISSUE TITLE
Install failed with go 1.18 on darwin

### ISSUE THREAD

**Issue by DonRenando** [sonatype-nexus-community/nancy#260]
Hello ! 

* **What are you trying to do?**
Upgrade  go to 1.18 and install nancy

* **What feature or behavior is this required for?**
Upgrade go from 1.17 to 1.18 on darwin and install nancy with : 
`go build -v -o=../bin/nancy github.com/sonatype-nexus-community/nancy`

* **How could we solve this issue? (Not knowing is okay!)**
Indirect dependency on an old version of x/sys that breaks when running go install with go 1.18. 
The fix appears to be to upgrade the dependency on x/sys. https://github.com/golang/go/issues/49219
with the latest version of golang.org/x/sys: v0.0.0-20220317061510-51cd9980dadf everything works again

The dependency come from go-sona-types
```
go mod graph | grep golang.org/x/sys@v0.0.0-20200519105757-fe76b779f299
github.com/sonatype-nexus-community/go-sona-types golang.org/x/sys@v0.0.0-20200519105757-fe76b779f299
```

An issue has been opened : https://github.com/sonatype-nexus-community/go-sona-types/issues/23 

* **Anything else?**
No 

Thank you

cc @bhamail / @DarthHater

**Comment by bhamail** [comment]
@DonRenando Thanks for the patch! I've released a new build of `nancy` (v1.0.33) that uses the patched `go-sona-types`. Could you please verify if this fixes the problem install problem on go v1.18?

**Comment by DonRenando** [comment]
thank you @bhamail !  the upgrade has fixed the problem on go v1.18.

**Comment by bhamail** [comment]
Fixed by https://github.com/sonatype-nexus-community/go-sona-types/pull/24
